js/node/node-backend.js

```diff
@@ -1,7 +1,8 @@
 import { isMainThread, parentPort, workerData, Worker } from 'worker_threads';
 import os from 'os';
 import wasm_ from '../../../_node_bindings/plonk_wasm.cjs';
-let filename = import.meta.url?.slice(7) ?? __filename;
+import { fileURLToPath } from 'url';
+let filename = fileURLToPath(import.meta.url);
```

Comment:
nice! this won't work in the commonjs version though - can you cover the case where `import.meta.url` is undefined and use `__filename` in that case? thanks

Reply:
Nice catch, I should considered this.